app/mailers/message_mailer.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 class MessageMailer < ApplicationMailer
+  default username: I18n.t('message_mailer.username')
```

Comment:
How does that work? I can’t find documentation on the `username` option anywhere…